commands/export_e2e_test.go

```diff
@@ -0,0 +1,462 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/utils"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestExportListCmdF() {
+	s.SetupTestHelper()
+	serverPath := os.Getenv("MM_SERVER_PATH")
+	importName := "import_test.zip"
+	importFilePath := filepath.Join(serverPath, "tests", importName)
+	exportPath, err := filepath.Abs(filepath.Join(*s.th.App.Config().FileSettings.Directory,
+		*s.th.App.Config().ExportSettings.Directory))
+	s.Require().Nil(err)
+
+	s.Run("no permissions", func() {
+		printer.Clean()
+
+		err := exportListCmdF(s.th.Client, &cobra.Command{}, nil)
+		s.Require().NotNil(err)
+		s.Require().Equal("failed to list exports: : You do not have the appropriate permissions., ", err.Error())
+		s.Require().Empty(printer.GetLines())
+		s.Require().Empty(printer.GetErrorLines())
+	})
+
+	s.RunForSystemAdminAndLocal("no exports", func(c client.Client) {
+		printer.Clean()
+
+		err := exportListCmdF(c, &cobra.Command{}, nil)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Empty(printer.GetErrorLines())
+		s.Equal("No export files found", printer.GetLines()[0])
+	})
+
+	s.RunForSystemAdminAndLocal("some exports", func(c client.Client) {
+		cmd := &cobra.Command{}
+		if c == s.th.LocalClient {
```

Comment:
Why do we need to set this flag explicitly for local tests?

Reply:
We don't. It was an imports only quirk. Removing :+1: